### PR TITLE
Added contributor guidelines

### PR DESCRIPTION
**Pull request by budhram**
Hi @nisusam 

Please check the contributor guidelines in English as well as Hindi as you intended. 
Tried my level best to get this.

Used https://www.google.com/inputtools/try/ and Google translate for Hindi translation.

### CODE REVIEW

File: CONTRIBUTING.hindi.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## परीक्षा मंच परियोजना मे योगदान कैसे करे
```
Comment (nisusam) [inline CONTRIBUTING.hindi.md]: `?` must be there in the end for every question raised

File: CONTRIBUTING.hindi.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* सुनिश्चित करें कि [मुद्दों](https://github.com/shikshaplus/pareeksha-manch/issues) के तहत गिटहब पर खोज करके **बग पहले ही रिपोर्ट नहीं किया गया** था |
+
+* यदि आप समस्या को हल करने में एक खुली समस्या नहीं ढूंढ पा रहे हैं, तो [एक नया खोलें](https://github.com/shikshaplus/pareeksha-manch/issues/new)। एक **शीर्षक और स्पष्ट विवरण**, जितना संभव हो उतना प्रासंगिक जानकारी शामिल करना सुनिश्चित करें, और एक **कोड नमूना** या **निष्पादन योग्य परीक्षण केस** जो अपेक्षित व्यवहार का प्रदर्शन नहीं कर रहा है |
+
+#### **क्या आपने एक पैच लिखा है जो एक बग को हल करता है ?**
+
+* पैच के साथ एक नया गिटहब पुल रिक्वेस्ट(पी.आर.) खोलें ।
+
+* सुनिश्चित करें कि पी.आर. विवरण समस्या और समाधान का स्पष्ट रूप से वर्णन करता है। लागू होने पर प्रासंगिक समस्या संख्या शामिल करें |
```
Comment (nisusam) [inline CONTRIBUTING.hindi.md]: ` ` missing in ```करता है।```

File: CONTRIBUTING.hindi.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+#### **क्या आपने रिक्‍त स्थान, कोड फॉरमेट को ठीक किया है, या पूरी तरह से कॉस्मेटिक पैच बनाया है ?**
+
+परिवर्तन जो प्रकृति में कॉस्मेटिक हैं और परियोजना की स्थिरता, कार्यक्षमता या टेस्टेबिलिटी के लिए कुछ भी महत्वपूर्ण मूल्य नहीं जोड़ते हैं, आमतौर पर स्वीकार नहीं किया जाएगा :wink: |
+
+#### **क्या आप एक नई सुविधा जोड़ने या मौजूदा एक को बदलने का इरादा रखते हैं ?**
+
+सुविधा या परिवर्तन से संबंधित प्रासंगिक जानकारी के साथ एक नया [गिटहब मुद्दा](https://github.com/shikshaplus/pareeksha-manch/issues/new) खोलें। हमें आशा है कि आप मुद्दे निर्माण के दौरान अधिकतर जानकारी प्रदान करेंगे :smile: |
+
+#### *परीक्ष मंच एक स्वयंसेवक प्रयास है । हम आपको [टीम](https://github.com/shikshaplus/pareeksha-manch/graphs/contributors) में शामिल होने के लिए प्रोत्साहित करते हैं | आशा करते हैं कि आप इस प्रयास में हमारा साथ देंगे |*
```
Comment (nisusam) [inline CONTRIBUTING.hindi.md]: `परीक्षा`

Comment (nisusam) [reply CONTRIBUTING.hindi.md]: `|` must be consistent throughout the `Hindi` guidelines.
Don't make it `italic`

File: CONTRIBUTING.hindi.md (hunk 1 above)
Comment (budhram) [reply CONTRIBUTING.hindi.md]: Not required like here https://github.com/rails/rails/blob/master/CONTRIBUTING.md#how-to-contribute-to-ruby-on-rails :smile:

File: CONTRIBUTING.hindi.md (hunk 2 above)
Comment (budhram) [reply CONTRIBUTING.hindi.md]: Ya got it thanks :)

File: CONTRIBUTING.hindi.md (hunk 3 above)
Comment (budhram) [reply CONTRIBUTING.hindi.md]: Sure, will make it italic. This part need to be italic to get highlighted from other texts.

**Comment by nisusam** [comment]
@budhram Thanks for the PR :+1:

**Pull request merged**